pdc/apps/auth/views.py

```diff
@@ -140,22 +140,26 @@ def user_profile(request):
     return render(request, 'user_profile.html', context)
 
 
+def _get_users_mapping_group(group, inactive_user):
+    users_set = {str(user.username) for user in get_user_model().objects.filter(groups__name=group)}
+    users_list = sorted(list(users_set - inactive_user))
+    return users_list
+
+
 def get_users_and_groups(resource_permission):
-    members = dict()
-    inactive_user = {user.username for user in models.User.objects.filter(is_active=False)}
-    superusers_set = {user.username for user in models.User.objects.filter(is_superuser=True)}
+    group_popover = OrderedDict()
+    inactive_user = {str(user.username) for user in models.User.objects.filter(is_active=False)}
+    superusers_set = {str(user.username) for user in models.User.objects.filter(is_superuser=True)}
     try:
         group_resource_permission_list = get_list_or_404(models.GroupResourcePermission,
                                                          resource_permission=resource_permission)
         groups_list = [str(obj.group.name) for obj in group_resource_permission_list]
-        users_set = {user.username for user in get_user_model().objects.filter(groups__name__in=groups_list)}
-        users_list = list(superusers_set.union(users_set) - inactive_user)
+        for obj in groups_list:
+            group_popover.setdefault('@' + obj, _get_users_mapping_group(obj, inactive_user))
```

Comment:
is this _get_users_mapping_group only used here, do we need to add a method for it?
and it seems the function for the method is to exclude the inactive_user from a group